examples/drag_and_drop/src/lib.rs

```diff
@@ -0,0 +1,198 @@
+use seed::{prelude::*, *};
+use web_sys::{self, DragEvent, Event, HtmlDivElement};
+// ------ ------
+//     Init
+// ------ ------
+
+fn init(_: Url, _: &mut impl Orders<Msg>) -> Model {
+    Model {
+        drop_zone: Default::default(),
+        drag_me_1: Default::default(),
+        drag_me_2: Default::default(),
+        who_is_getting_dragged: None,
+    }
+}
+
+// ------ ------
+//     Model
+// ------ ------
+
+struct Model {
+    drop_zone: ElRef<HtmlDivElement>,
+    who_is_getting_dragged: Option<u8>,
+    drag_me_1: ElRef<HtmlDivElement>,
+    drag_me_2: ElRef<HtmlDivElement>,
+}
```

Comment:
```suggestion
struct Model {
    drop_zone: ElRef<HtmlDivElement>,
    drag_me_1: ElRef<HtmlDivElement>,
    drag_me_2: ElRef<HtmlDivElement>,
    who_is_getting_dragged: Option<u8>,
}
```
To group similar items and to make the field order corresponding with the order in `Model` creation in `fn init`.